Models/GumbelModels/MCTSCtree.py

```diff
@@ -33,6 +32,7 @@ def __init__(self, model_config: None) -> None:
         # Get the default configuration.
         self.model_config = model_config
         self.discount_factor = config.DISCOUNT
+        import core.gumbelctree.gmz_tree as tree_gumbel_muzero

```

Comment:
I'm pretty sure this will break this method if it is in the init but I can take care of this.